### DISCUSSION TITLE
Way to limit number of blocks per group

### DISCUSSION THREAD

**Discussion by PapiCapi** [GriefPrevention/GriefPrevention#924] [Q&A] [answered]
I checked literally all pages of the wiki and checked a few pages of the spigot thread but didn't found anything.

I was asking myself if there was a way to set different limits of blocks ( "Max Accrued Claim Blocks" ) for different ranks/groups.
For exemple: 
member : 5000 blocks
vip : 9000 blocks

Thanks in advance

**Comment by Jikoo** [comment, marked as answer]
https://github.com/TechFortress/GriefPrevention/wiki/Addons#gpaccruals
